sdk/core/azure/HISTORY.rst

```diff
@@ -1,8 +1,9 @@
 Disclaimer
 ++++++++++
 
-Starting azure 3.0.0, we don't provide anymore a ChangeLog for the bundle package. Please refer to the ChangeLog
-of each individual package.
+Starting v5.0.0, this package cannot be installed anymore, please install individual packages.
```

Comment:
We should add an issue that explains in a bit more depth why we deprecated this package.

Reply:
https://github.com/Azure/azure-sdk-for-python/issues/10646

Need review :)